parser_test.go

```diff
@@ -1714,6 +1714,72 @@ func TestSelectStatement(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:     "case no else",
```

Comment:
Test for state [163](https://github.com/tablelandnetwork/sqlparser/pull/5/commits/ee33f5b35e66c8489e2a21809bfdde416e4c20cf#diff-cfe1c0871e20901b761bf77cb4f881b71214777ece99315a6b3fb7124ac997f9R1363)